mobile-app/lib/models/learn/curriculum_model.dart

```diff
@@ -202,7 +202,7 @@ class Block {
     return Block(
       superBlock: SuperBlock(
         dashedName: superBlockDashedName,
-        name: superBlockName,
```

Comment:
Since this variable is replaced, we can remove it from the function arguments as it's not used anywhere else

TODO: Add a linter rule to catch unused arguments/parameters